docs/integrations/sources/surveymonkey.md

```diff
@@ -1,72 +1,103 @@
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
 # SurveyMonkey
 
-This page guides you through the process of setting up the SurveyMonkey source connector.
+This guide contains instructions for setting up an Airbyte Cloud or Open Source connector for SurveyMonkey.
 
 :::note
 
-OAuth for Survey Monkey is officially supported only for the US. We are testing how to enable it in the EU at the moment. If you run into any issues, please [reach out to us](mailto:product@airbyte.io) so we can promptly assist you.
+OAuth for Survey Monkey is officially supported only for the US. Airbyte is testing how to enable it in the EU. If you run into issues, [reach out to support](mailto:product@airbyte.io).
```

Comment:
Per Google's style guide, removing "we" and "please". Biasing towards conciseness where possible.